examples/weaviate/personalized_description_with_weaviate_and_gemini_api.ipynb

```diff
@@ -0,0 +1,963 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    genai.configure(api_key='gemini-api-key') # gemini api key
</pre><p> </p><p>Once you set up the common API key cell, you can reuse the same key here.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>